README.md

```diff
@@ -153,6 +154,25 @@ var instance = pino({
 }
 ```
 
+<a name="child"></a>
+### logger.child(opts)
```

Comment:
not sure that "opts" is the right word? bunyan docs refers to them as "bound fields" so maybe "fields" would be a better description? or "bindings"?